### DISCUSSION TITLE
Alternatives to add-on with bundled product

### DISCUSSION THREAD

**Discussion by ginkel** [pretix/pretix#4219] [Q&A]
Hi there,

this is somewhat a continuation of https://github.com/pretix/pretix/discussions/3912. The solution proposed there is working like a charm for regular products with the tax-privileged product as bundled product.

Unfortuantely, we hit one corner-case where we would need to apply this to an add-on product when an adult books an extra seat for his/her child, which currently seems to be unsupported ("You cannot add bundles to a product that is only available as an add-on itself.").

So in the end we'd like to come up with the following (simplified) structure:

Conference ticket (1) - 123,45 € (19% VAT)
- Extra child (add-on to conference ticket) (2) - 23,45 € (19% VAT)
  - Overnight stay with full board (for the child) (3) - 12,34 € (0% VAT)

I can't just merge 2 and 3 due to the differing VAT rate.

Any ideas how we might accomplish this?

Thanks,
Thilo

**Comment by raphaelm** [comment]
I don't think we have a tool for that

**Reply by ginkel** [reply]
Hm... I'll see if I can figure out any workaround. I guess the limitation why this kind of nesting is unsupported is the UI, right (and not the ORM model)?

**Reply by raphaelm** [reply]
Not the ORM, no, but the nesting is not supported in loooots of code paths beyond the UI, it's just out of scope.

**Reply by ginkel** [reply]
Thanks for the feedback!
